backend/models/dynamodb/Entity.js

```diff
@@ -0,0 +1,76 @@
+
+import DynamoDB from "@klaudsol/commons/lib/DynamoDB"
+import { DYNAMO_DB_TYPES, DYNAMO_DB_INDEXES, DYNAMO_DB_TABLE } from "@/constants";
+
+export default class Entity {
+  // Gets all items where entity = entity_type_slug (eg. tech)
+  // and if type = content
+  // Sample: api/tech
+  // TODO: Add pagination
+  // TODO: Add filtering
+  // TODO: Add search
+  static async whereAll({ entity_type_slug }) {
```

Comment:
I think we need to make the method names specific to the access patterns. This would easily tell the developers we do not have a generic "WHERE" statement similar to a relational database. Each function would be on a per-usage basis.

```suggestion
  static async whereEntityTypeSlug(entity_type_slug) {
```